src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carPriceDay = 40;
+  const discountSevenDays = 50;
+  const discountTwoDays = 20;
```

Comment:
```suggestion
  const bigDiscount = 50;
  const basicDiscount = 20;
```
It is better not to link the variable to values that can be changed later